src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/MethodTypeInference.cs

```diff
@@ -135,6 +135,19 @@ private enum Dependency
         private readonly HashSet<TypeWithAnnotations>[] _exactBounds;
         private readonly HashSet<TypeWithAnnotations>[] _upperBounds;
         private readonly HashSet<TypeWithAnnotations>[] _lowerBounds;
+
+        // https://github.com/dotnet/csharplang/blob/main/proposals/csharp-9.0/nullable-reference-types-specification.md#fixing
+        // If the resulting candidate is a reference type or a nonnullable value type and all of the
+        // exact bounds or any of the lower bounds are nullable value types, nullable reference
+        // types, null or default, then ? is added to the resulting candidate, making it a nullable
+        // value type or reference type.
```

Comment:
THe "nullable value type" portion seems wrong. We're not changing those

Reply:
Changed the quote here to align with dotnet/csharplang#5138.